mpdev/internal/resources/deployment_manager.go

```diff
@@ -113,7 +113,7 @@ func (dm *DeploymentManagerAutogenTemplate) Test(registry Registry, dryRun bool)
 }
 
 func (dm *DeploymentManagerAutogenTemplate) runAutogen(registry Registry, inputDir string) error {
-	autogenImg := "gcr.io/cloud-marketplace-tools/dm/autogen"
+	autogenImg := "$ARTIFACT_REGISTRY/cloud-marketplace-tools/dm/autogen"
```

Comment:
The user will likely have to replace the whole path to the image with their own, so the "cloud-marketplace-tools/dm/" part also needs to be replaced.